networkx/algorithms/vitality.py

```diff
@@ -70,13 +70,13 @@ def closeness_vitality(G, weight=None):
         # remove edges connected to node n and keep list of edges with data
         # could remove node n but it doesn't count anyway
         if multigraph:
-            edges = G.edges(n,data=True,keys=True)
+            edges = list(G.edges(n,data=True,keys=True))
```

Comment:
Spaces here (again, not sure if this is the right time to make these changes or not).